packages/medusa/package.json

```diff
@@ -41,15 +41,15 @@
     "serve": "node dist/app.js",
     "test": "jest --silent --bail --maxWorkers=50% --forceExit"
   },
+  "peerDependencies": {
+    "@medusajs/medusa-cli": "*"
+  },
```

Comment:
thought: would it make sense to make `@medusajs/medusa-cli` a dependency of `@medusajs/framework`. we want the bin to always be available when the framework package is installed so think it'd be fair.

Reply:
I see now that it already is a dependency in @meduasjs/framework so is the peerDependency necessary?

Reply:
It is like that. `@medusajs/medusa-cli` is a dependency of `@medusa/framework`, but just a peerDependency at `@medusajs/medusa`, that already have the framework as a dependency.

Reply:
At this moment it wouldn't be required because I removed the `require("@medusajs/medusa-cli")` from medusa, but maybe we'll need some other things exported from the cli later, keeping it as a peerDependency at least we have no risk of requiring a different version of it here